root/help.md

```diff
@@ -0,0 +1,41 @@
+% DHCP-CLIENT (1) Container Image Pages
+% Karsten Hopp
```

Comment:
I thought of MAINTAINER field as more of a person who is maintaining the actual software, not the person who wrote the docs. And right now I can't find any specification that would back either opinion. Should we decide then what would be more appropriate and add that to guidelines (best practices)? Or did I miss something and it is actually resolved already somewhere?